From 791e420360669d55b7f90ca0c6d61b10e4992aec Mon Sep 17 00:00:00 2001
From: Bhaskar Chowdhury <unixbhaskar@gmail.com>
Date: Fri, 14 Aug 2026 15:21:37 +0530
Subject: [PATCH 13996/15333] Doc: admin-guide: pm: Remove unnecessary
 backticks and fix a spell

One,
Removed the superfluous backticks in the sentence.

And second,

s/peferred/preferred/

Signed-off-by: Bhaskar Chowdhury <unixbhaskar@gmail.com>
Acked-by: Randy Dunlap <rdunlap@infradead.org>
Acked-by: Mario Limonciello (AMD) <superm1@kernel.org>
Signed-off-by: Jonathan Corbet <corbet@lwn.net>
Message-ID: <20260814095716.2905589-1-unixbhaskar@gmail.com>
---
 Documentation/admin-guide/pm/amd-pstate.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/admin-guide/pm/amd-pstate.rst b/Documentation/admin-guide/pm/amd-pstate.rst
index a95e2ebce005..f190aaa5281d 100644
--- a/Documentation/admin-guide/pm/amd-pstate.rst
+++ b/Documentation/admin-guide/pm/amd-pstate.rst
@@ -472,7 +472,7 @@ update the core ranking and set the cpu's priority.
 Kernel Parameters
 -----------------
 
-``amd-pstate`` peferred core`` has two states: enable and disable.
+``amd-pstate`` preferred core has two states: enable and disable.
 Enable/disable states can be chosen by different kernel parameters.
 Default enable ``amd-pstate`` preferred core.
 
-- 
2.55.0

